docs/tools_user_guide.md

```diff
@@ -8,14 +8,18 @@ MLAgility's tools currently support the following combinations of runtimes and d
 
 <span id="devices-runtimes-table">
 
-| Device Type | Device arg | Runtime      | Specific Devices                                         |
-| ----------- | ---------- | ------------ | -------------------------------------------------------- |
-| Nvidia GPU  | nvidia     | TensorRT     | Any Nvidia GPU supported by TensorRT >= 8.5.2            |
-| x86 CPU     | x86        | ONNX Runtime | Any Intel or AMD CPU supported by ONNX Runtime >= 1.13.1 |
-| Groq        | groq       | GroqFlow     | GroqChip1                                                |
-
+| Device Type | Device arg | Runtime                                                            | Runtime arg                | Specific Devices                              |
+| ----------- | ---------- | ------------------------------------------------------------------ | -------------------------- | --------------------------------------------- |
+| Nvidia GPU  | nvidia     | TensorRT<sup>†</sup>                                               | trt                        | Any Nvidia GPU supported by TensorRT          |
+| x86 CPU     | x86        | ONNX Runtime<sup>‡</sup>, Pytorch, Pytoch 2.x Compiled<sup>*</sup> | ort, torch, torch_compiled | Any Intel or AMD CPU supported by the runtime |
+| Groq        | groq       | GroqFlow                                                           | groqflow                   | GroqChip1                                     |
```

Comment:
Now that we dont have parity between remote and local benchmarking, do we want to include which runtimes supports cloud and which ones don't?

Reply:
Added!